addons/website/tests/test_ui.py

```diff
@@ -459,6 +459,9 @@ def test_15_website_link_tools(self):
     def test_16_website_edit_megamenu(self):
         self.start_tour(self.env['website'].get_client_action_url('/'), 'edit_megamenu', login='admin')
 
+    def test_16_2_website_megamenu_active_nav_link(self):
```

Comment:
we stopped using numbers in our test names, you can simply call it `test_website_megamenu_active_nav_link` ;)